### DISCUSSION TITLE
Can you explain these sliders please?

### DISCUSSION THREAD

**Discussion by gerroon** [Acly/krita-ai-diffusion#1158] [Q&A] [answered]
How do these two work together? thnx

![image](https://github.com/user-attachments/assets/68e3d42f-e744-4b30-8695-e218eb5a3b53)

**Comment by minsky91** [comment, marked as answer]
The slider below (called _Denoising strength_ In other Stable Diffusion tools) is to control the _general_ strength of the AI refinement generation: the lower the value, the closer the output will resemble the input image, and the higher the value, the more the output will diverge from the input. A value of 95% will cause generation of an image (or of a selected region) in a new layer that will only minimally resemble the currently active layer’s content. 

The top slider (called Weight of a controlnet in other Stable Diffusion tools) is to adjust the strength (influence) of the _control layer_ you have created (Line Art, in this case). In the Krita AI plugin, you are allowed to create a practically unlimited number of control layers for the currently active image/layer, so naturally you also want to control their relative strength, which is what these sliders are for. 
 
By playing with each slider separately, you learn how it influences the generated output; adjusting them simultaneously before generation will only confuse you. I would suggest starting with a fixed low-to-medium general Strength (like 40%) without any control layer and see how this parameter affects the output, in a batch of say 10 images. Only then I’d proceed to creating a control layer and playing with its values. (Usually the default value is good enough.)
